index.html

```diff
@@ -254,16 +259,28 @@ <h1 id="abt_mslf">About Me</h1>
       </ul>
     </ul>
   </section>
-
   <!-- Contact  -->
   <section class="contact">
     <h1 id="headerrr">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h1>
-    <input type="text" placeholder="Full Name" id="name">
-    <input type="text" placeholder="Email Address" id="email">
-    <input type="text" placeholder="Write me something..." id="para">
-    <button id="button1">Get in touch</button>
-    <hr>
+    <form action="https://formspree.io/f/mqkorarr" method="post">
```

Comment:
- [ ] Great effort working on the contact section of the portfolio project, but currently, i noticed that i could not submit the form after i have inserted all the necessary data. I would kindly suggest you work on the `formspree` and be sure the id is properly used so as to aid submission of data inserted by a user.

Video claim 👇


https://github.com/Rdzaman20000/Portfolio/assets/78634841/4ff0c6cc-63c6-4dff-9570-4874ea6f0df5

Reply:
@brainconnect93 I have implemented the button for the mobile version and not the desktop one yet. so kindly try to submit it using the mobile version e.g reducing .  Thanks